src/App.js

```diff
@@ -14,11 +15,59 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    choosenItem: '',
+  };
+
+  chooseItem = (event) => {
+    this.setState({ choosenItem: event.target.innerText });
+  };
+
+  clearSelection = (event) => {
```

Comment:
Якщо не використовуєш event то не залишай лишні параметри..
```suggestion
  clearSelection = () => {
```